lib/display.ts

```diff
@@ -0,0 +1,175 @@
+import { createFromJSON, DepGraph } from "@snyk/dep-graph";
+import chalk from "chalk";
+
+import { Issue, Options, ScanResult, TestResult } from "./types";
+
+export async function display(
+  scanResults: ScanResult[],
+  testResults: TestResult[],
+  errors: string[],
+  options?: Options,
+): Promise<string> {
+  const testingPath = options?.path;
+
+  const result: string[] = [];
+  if (options && options.path) {
+    const prefix = chalk.bold.white("\nTesting " + testingPath + "...\n\n\n");
+    result.push(prefix);
+  }
```

Comment:
I think this line won't even have the chance to be seen by the user, the whole `display` function seems like one big synchronous block so I think maybe we don't need this line?

Reply:
I still think `const prefix = chalk.bold.white("\nTesting " + testingPath + "...\n\n\n");` won't get displayed to the user like for example when the spinner is waiting for the results, do we need this line?

Reply:
I agree! But I just copied it from the way it's being done in the CLI. We can remove it. Wdyt?